### DISCUSSION TITLE
Going multilanguage?

### DISCUSSION THREAD

**Discussion by Ndrinta** [nathanmarlor/foxess_modbus#453] [General]
Hello everyone, I have noticed that an increasing number of Italians are joining the Facebook groups and this forum. Although my Facebook may be biased, I am sure that other nationalities are also joining. I am here to ask if anyone knows of a way to have multiple translations for the same page. Does Git or the Wiki section support such a feature? I am willing to start translating the most important pages to clear up any confusion that may arise from the output generated by the auto-translation tools. Technical jargon is often mistranslated, so I believe that this would be a helpful step to take.

**Comment by canton7** [comment]
I'd been wondering when we might start needing to internationalise... HA has some support for the integration itself, although noone's actually asked for it yet.

I don't believe there's any good way of doing translations within github - most people move to an external docs system (eg readthedocs), and handle translations though a third-party platform.

And of course, all of that is effort... I'd be interested to hear if anyone actually wants translations first! I guess setting up HA etc in the first place requires some level of technical literacy, which necessarily these days comes hand in hand with technical English.
